diff --git a/ios/DomWebView.swift b/ios/DomWebView.swift
index 7a38d9f98bf8c37853b777dbf945a13d46560e30..284dbb85ac2941fe359f4a5d58bf97b21ec915ef 100644
--- a/ios/DomWebView.swift
+++ b/ios/DomWebView.swift
@@ -2,6 +2,7 @@
 
 import ExpoModulesCore
 import WebKit
+import React
 
 internal final class DomWebView: ExpoView, UIScrollViewDelegate, WKUIDelegate, WKScriptMessageHandler {
   // swiftlint:disable implicitly_unwrapped_optional
